src/main/java/com/dougnoel/sentinel/files/TestFile.java

```diff
@@ -0,0 +1,33 @@
+package com.dougnoel.sentinel.files;
+
+import com.dougnoel.sentinel.strings.SentinelStringUtils;
+import com.dougnoel.sentinel.system.DownloadManager;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.nio.file.Path;
+
+public abstract class TestFile extends File{
+    /**
+     * Creates a TestFile from the DownloadManager's latest download and immediately checks that it exists. If it doesn't, an exception is thrown.
+     *
+     * @throws FileNotFoundException in the case that a file is not found at the path given by the DownloadManager,
+     * or if a file has not been downloaded in this test session.
+     */
+    protected TestFile() throws FileNotFoundException {
+        this(DownloadManager.getMostRecentDownloadPath());
+    }
```

Comment:
This constructor isn't used. I'd like to see a test for this even if it's just injecting the test file into the correct value in the Download Manager.

Reply:
This will probably be covered if you do this in the CsVFile constructor.